source/locale/pt_BR/LC_MESSAGES/using-web-interface.po

```diff
@@ -463,6 +463,39 @@ msgstr ""
 "Note que, em cada etapa, são exibidos o status e o resultado da aplicação"
 " da imagem. "
 
+#: ../../source/using-web-interface.rst:247
```

Comment:
Just a question ... Are you adding manually,by hand? 
If so I believe it would cause some problems ....
Are you generating the html in pt BR and checking everything?

Reply:
I only ask because `msgid` is a single line when use `make gettext`  it breaks the lines.

Reply:
But if you generate the html in pt BR and It works that already enough ...